dags/main_summary.py

```diff
@@ -95,7 +95,13 @@
         dataset="main_summary",
         dataset_version="v4",
         gke_cluster_name="bq-load-gke-1",
-        ),
+        bigquery_dataset="telemetry_raw",
+        cluster_by=["sample_id"],
+        drop=["submission_date_s3"],
+        replace=[
+            "submission_date_s3 AS submission_date",
```

Comment:
Does a field named `submission_date` already exist? If not, then this will fail as a REPLACE clause. It may need to come before the `* REPLACE`, which may require an additional new parameter.

Reply:
yes, it does already exist. for now, this will work. for other tables hiding `submission_date_s3` may required another parameter, correct.

Reply:
https://github.com/mozilla/parquet2bigquery/pull/30